src/commands/invites/schema/delete.ts

```diff
@@ -0,0 +1,47 @@
+import { type ArgumentsCamelCase, type Argv, type CommandModule } from 'yargs'
+
+import { apiCommand, apiCommandBuilder, type APICommandFlags } from '../../../lib/command/api-command.js'
+
+import { chooseSchemaInvitationFn } from '../../../lib/command/util/schema-invites-choose.js'
+
+
+export type CommandArgs =
+	& APICommandFlags
+	& {
+		id?: string
+		schemaApp?: string
+	}
+
+export const command = 'invites:schema:delete [id]'
+
+const describe = 'delete a Schema App invitation'
+
+const builder = (yargs: Argv): Argv<CommandArgs> =>
+	apiCommandBuilder(yargs)
+		.positional('id', { describe: 'Schema App invitation id', type: 'string' })
+		.option('schema-app', { describe: 'Schema App id', type: 'string' })
+		.example([
+			[
+				'$0 invites:schema:delete',
+				'choose the invitation to delete from a list',
+			],
+			[
+				'$0 invites:schema:delete --schema-app e077532e-3c0f-4f3b-a1d8-029115ee1602',
+				'select an invitation to delete from list, for the specified schema app',
```

Comment:
A bit too long, maybe, but what about "choose the invitation to delete from a list of invitations for the specified schema app"?

Reply:
Yeah, that's definitely better and not any longer than some of our other examples so I've used your wording. (And I also capitalized Schema App.)